lib/tasks/publishing_api.rake

```diff
@@ -84,4 +84,35 @@ namespace :publishing_api do
     )
     puts "Links patched for root page..."
   end
+
+  desc "Update redirect for an archived tag"
+  task :update_redirect, %i[content_id new_redirect_path] => :environment do |_task, args|
+    unless args[:new_redirect_path].present? && args[:new_redirect_path].starts_with?("/")
+      raise "new_redirect_path is missing or invalid"
+    end
+
+    if Services.content_store.content_item(args[:new_redirect_path])
```

Comment:
Hi, is it helpful to extracting this out to a Boolean function so that we know what the content_item function is doing with the new_redirect_path?

Reply:
Thanks for the review Siva!

This is just a quick way for us to see if a valid redirect path has been provided. It could just be a `Services.content_store.content_item(args[:new_redirect_path])` without being called as part of an `if`. It's just so that if the content store does not have a valid content item, this line will raise an error and halt the rake task.

Given more time, and a longer term need for this task, I would have liked to create a `redirect` object and include the [valid_govuk_path_validator](https://github.com/alphagov/collections-publisher/blob/main/app/validators/valid_govuk_path_validator.rb). But as time is short and we are removing the code once the task has been run, I've just copied the validation code into this rake task as a quick fix.

Hope that makes sense and you're happy to leave as it is?

Reply:
Sounds good, thank you!